### ISSUE TITLE
Guide: How to search language specific torrents

### ISSUE THREAD

**Issue by Francesco980** [elgatito/script.elementum.burst#386]
When I do a search for a movie or TV series when the search finishes I never find the movie or TV series in my language which would be Italian, how can I solve this problem to easily find movies and TV series in my language?

**Comment by antonsoroko** [comment]
@elgatito can we please open this issue (and keep it open) and rename it to "Guide: How to search language specific torrents" so other people can find this info. thanks!

**Comment by Francesco980** [comment]
In advanced settings of Elementum burst in additional keyword filters I don't understand what should I write to search torrent in my language should I write so ita. _ eng. _ or ita, _ eng, _ I don't understand how I should write it, and I can't find TV series in Korean or Japanese even if the title of the TV series is written in Italian or English.

**Comment by Francesco980** [comment]
@antonsoroko In advanced settings of Elementum burst in additional keyword filters I don't understand what should I write to search torrent in my language should I write so ita. _ eng. _ or ita, _ eng, _ I don't understand how I should write it, and I can't find TV series in Korean or Japanese even if the title of the TV series is written in Italian or English.

**Comment by antonsoroko** [comment]
@Francesco980 

TLDR: if you are inexperienced user - please use **only** italian trackers/providers and enable translation to italian. do what i wrote in 1 and 2.

long answer:
note that "advanced settings" are only for advanced users. if you do not know how italian pirates name their torrents - then this option is not for you. if you want to became advanced user - please do research and learn how italian torrents are usually named (e.g. what words they add to names). then you will know what words to use.
but again - this is advanced setting, thus you must know what you do. it will filter out (remove) results that do not have words that you specified.

>I don't understand how I should write it

it has description about format! and it is translated to itialian - i checked via translator - translation is correct.
Use comma (comma is `,` not `.`) as separator. Use `_TEST_` if you want to check torrent name against " TEST " (with spaces around word), otherwise just write "TEST" - but note that this can match words like "TESTERS" or "PROTEST". (Unfortunately, despite the fact it also says about dots - it will not match dots, only spaces. Looks like a bug.)

>I can't find TV series in Korean or Japanese even if the title of the TV series is written in Italian or English.

i guess you use wrong torrent trackers. use torrent trackers that have videos that you want in the language you want.

**Comment by Francesco980** [comment]
@antonsoroko But why when I do a search for a movie or TV 0series in the search results I no longer see 1337x? Is there any way when I do a search in the results there is this site?

**Comment by antonsoroko** [comment]
@Francesco980 see https://github.com/elgatito/script.elementum.burst/issues/240#issuecomment-1918084633
they started to use aggressive anti-bot protection and burst is a bot. so 1337x sometimes works for some people (it works for me), but usually it does not work. If it does not work for you - just disable it.
To test if it really does not work for you (not because you misconfigured your kodi/elementum/burst): just use "search" on main page of elementum and search for some popular video in english, of course you need to remove all "Additional keyword filters" in burst, if you already set them, so they will not affect test results.)

Anyway, 1337x is not italian tracker but, so it has all languages.

I strongly recommend you to use italian trackers to avoid all this fuss - this is the easiest and most robust solution for inexperienced users. as i can see we have ilcorsoronero (but you need to enable "allow results with 0 seeds" in settings for that tracker) and ilcorsoroblu. If you know basic HTML - you can add more.

But if for some strange reason you want to use international trackers to find only italian torrents - then set enable "Additional keyword filters (comma-separated)" and set "Require" to "ITA" or similar word that italian torrents have in their name (without quotes obviously). See my explanation about Accept/Block/Require above.
Also it is better to disable "Use language preference from Kodi when a translation is available" for international trackers, since they use english usually.

Alternatives to 1337x: magnetdl, bitsearch, Torrentz2K, thepiratebay (they recently changed their ui layout, will be fixed in #394), torrentgalaxy (although recenty they also started to block bots), and many other tracker - see settings.

Also, if you want to search video titles translated to italian (e.g. you found tracker that uses names translated to italian) - you need to set your kodi language to italian, otherwise burst will not know what language to use for search (if you enabled "Use language preference from Kodi when a translation is available" option as i told you before). But again, it makes sense only if you use italian trackers/providers or trackers that have torrents with italian titles.

**Comment by antonsoroko** [comment]
@elgatito i updated the guide https://github.com/elgatito/script.elementum.burst/issues/386#issuecomment-1973041656 
could you please "pin" this issue? thanks.

**Comment by elgatito** [comment]
> @elgatito i updated the guide https://github.com/elgatito/script.elementum.burst/issues/386#issuecomment-1973041656 
> could you please "pin" this issue? thanks.

But this issue is already pinned

**Comment by antonsoroko** [comment]
@elgatito my bad. looks like sometimes github does not show me pinned issues at all. some bug maybe.
